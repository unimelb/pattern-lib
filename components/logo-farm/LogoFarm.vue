<template>
  <section-wrap>
    <div class="logo-farm">
      <a
        v-for="(item, index) in data"
        :key="index"
        :href="item.url"
        class="logo-farm__link">
        <img
          :alt="item.alt"
          :src="item.src"
          class="logo-farm__image"
        >
      </a>
    </div>
  </section-wrap>
</template>

<script>
export default {
  props: {
    data: {
      type: Array,
      default: () => [],
    },
  },
};
</script>
