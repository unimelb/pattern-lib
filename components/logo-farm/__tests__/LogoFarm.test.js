import {
  shallow,
} from 'vue-test-utils';
import {
  toHaveNoViolations,
} from 'jest-axe';
import LogoFarm from '../LogoFarm.vue';


expect.extend(toHaveNoViolations);

describe('LogoFarm', () => {
  it('should match snapshot', () => {
    const result = shallow(LogoFarm).element;
    expect(result).toMatchSnapshot();
  });
});
