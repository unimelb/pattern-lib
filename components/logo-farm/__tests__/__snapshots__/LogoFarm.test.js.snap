// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogoFarm should match snapshot 1`] = `
<section-wrap>
  <div
    class="logo-farm"
  />
</section-wrap>
`;
